xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -612,6 +612,14 @@ extension DetailStackAction {
             return .requestUpdateAudience(address, audience)
         case let .requestAssignNoteColor(address, color):
             return .requestAssignNoteColor(address, color)
+        case let .requestQuoteInNewDetail(address):
+            return .pushDetail(
+                .editor(
+                    MemoEditorDetailDescription(
+                        fallback: "\(address.markup)\n\n"
+                    )
+                )
+            )
```

Comment:
This results in an untitled editor, preloaded with the link and the cursor autofocused in a new block. 

There is one UX rough edge, if you back out of the new editor this opens, then the app will automatically save the note... which creates a "garbage" note. Perhaps we should have some clever logic that never saves a note unless it contains something other than / in addition to a single slashlink?